releases/Gemfile

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "aws_sdk"
+gem "bunny"
+gem "concurrent-ruby"
+gem "dalli"
+gem "delayed_job"
+gem "ethon"
+gem "excon"
+gem "faraday"
+gem "grape"
+gem "graphql"
+gem "http"
+gem "httpclient"
+gem "koala"
+gem "lmdb"
+gem "mongo"
+gem "mysql2"
+gem "pg"
+gem "que"
+gem "racecar"
+gem "rack"
+gem "rails"
+gem "rake"
+gem "rdkafka"
+gem "redis"
+gem "resque"
+gem "rest-client"
+gem "rspec"
+gem "ruby-kafka"
+gem "sidekiq"
+gem "sinatra"
+gem "trilogy"
+
+gem "opentelemetry-api"
+gem "opentelemetry-sdk"
+gem "opentelemetry-resource_detectors"
+
+Dir["../propagator/**/version.rb"].each do |f|
+  name = f.match(/propagator\/(\w+)\/lib/)[1]
+
+  version = File.read(f).match(%r|VERSION = '([\d\.]+)'|)[1]
+  gem "opentelemetry-propagator-#{name}", version
+end
+
+Dir["../instrumentation/**/version.rb"].each do |f|
+  name = f.match(/instrumentation\/(\w+)\/lib/)[1]
+
+  version = File.read(f).match(%r|VERSION = '([\d\.]+)'|)[1]
+  gem "opentelemetry-instrumentation-#{name}", version
+end
```

Comment:
I wonder if you could do something similar to install the instrumented gems? (such as `gem 'mysql2'`) 

This might make it easier to keep the Gemfile up to date so that we don't have to remember to update this Gemfile each time a new library is instrumented.

Reply:
I would like that yes, however for whatever reason there is an idiosyncratic naming pattern for gems like `activesupport` and `opentelemetry-instrumentation-active_support` so it would likely result in a list of exceptions. :(